openmoc/numpy_typemaps.i

```diff
@@ -75,3 +75,9 @@
 
 /* The typemap used to match the method signature for Solver::setFluxes */
 %apply (FP_PRECISION* INPLACE_ARRAY1, int DIM1) {(FP_PRECISION* in_fluxes, int num_fluxes)}
+
+/* The typemap used to match the method signature for
+ * Geometry::getSpatialDataOnGrid */
+%apply (double* IN_ARRAY1, int DIM1) {(double* grid_x, int num_x)}
+%apply (double* IN_ARRAY1, int DIM1) {(double* grid_y, int num_y)}
+%apply (int* ARGOUT_ARRAY1, int DIM1) {(int* domains, int num_xy)}
```

Comment:
I'm open to using `std::vector`, but since I don't know how to, I went with NumPy typemaps. Do we have any examples which use `std::vector`? Is this documented in our Developer's Guide as are the NumPy typemaps? If not, I would opt to leave this as is for now in lieu of an overhaul of our NumPy typemaps to Python lists / `std::vectors` for this method as well as many others in the future.